### ISSUE TITLE
Null Character

### ISSUE THREAD

**Issue by gsugambit** [crysislinux/smart_websocket_client#2]
The websocket framework im using (STOMP in spring framework) wants me to send a NULL character '/u0000' or '\0' after my messages. Currently, I cannot see a way to do this with this tool. Is it possible?

**Comment by crysislinux** [comment]
no, it doesn't support that. It only support send a javascript string at the moment.
